### DISCUSSION TITLE
look left not working

### DISCUSSION THREAD

**Discussion by branaway** [pixiv/three-vrm#1143] [Help]
const lookleft = this.vrm.expressionManager.getExpressionTrackName( THREE_VRM.VRMExpressionPresetName.LookLeft );

It would return null.

The model is a custom made by an external artist if it's relevant. 

three-arm 1.0.6 with threejs 146

**Comment by branaway** [comment]
it would return undefined t be precise.

**Comment by tejasnayak25** [comment]
I'm working with something similar. In fact, I have issues setting the look left, look right, surprised, lookup, look down while other expressions are just working fine.

**Comment by branaway** [comment]
I cannot rule out the possibility that the model from VRoid studio is faulty thou.

**Comment by 0b5vr** [comment]
Moving to Discussions since it's an avatar specific issue

**Comment by 0b5vr** [comment]
VRoid Studio models doesn't have Look expressions, since the eye movement is manipulated by bones instead.
You would want to use LookAt API instead.

See the `lookat.html` example in three-vrm:
https://github.com/pixiv/three-vrm/blob/v1.0.6/packages/three-vrm/examples/lookat.html

**Reply by tejasnayak25** [reply]
What about surprised expression then?

**Reply by 0b5vr** [reply]
What do you mean?

Eye controls are LookAt's duty, facial expressions are Expression's duty.

**Reply by tejasnayak25** [reply]
Surprised and Neutral don't work too
